assets/less/reportwidgets.less

```
@@ -0,0 +1,308 @@
+                }
+
+                &:before {
+                    margin-right: 2px;
+                }
+            }
+        }
+    }
+
+
+
+    //
```

Stylelint (17.11.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)